qlib/pagetable.rs

```diff
@@ -364,6 +417,13 @@ impl PageTables {
         flags: PageTableFlags,
         pagePool: &Allocator,
     ) -> Result<bool> {
+        #[cfg(feature = "cc")]
+        if ENABLE_CC.load(Ordering::Acquire) {
```

Comment:
I was confused earlier because of the ENABLE_CC naming. I think in this context it's a flag variable that means "whether CC is enabled", the `ENABLE_CC` name sounds like a (rather static) configuration term.

Reply:
Yes, it is configurable via config file now